src/components/organisms/PluginManager/PluginInstallModal.tsx

```diff
@@ -85,12 +138,22 @@ function PluginInstallModal(props: {isVisible: boolean; onClose: () => void}) {
           See Plugin&apos;s documentation for more information.
         </Button>
       </p>
-      <Input placeholder="Enter Plugin URL" defaultValue={pluginUrl} onChange={e => setPluginUrl(e.target.value)} />
-      {errorMessage && (
-        <div>
-          <p>{errorMessage}</p>
-        </div>
-      )}
+      <Form layout="vertical" form={pluginForm} onFinish={onFinish} initialValues={() => formValues}>
+        <Form.Item
+          name="pluginUrl"
+          label="URL"
+          required
+          tooltip="pluginUrl"
```

Comment:
perhaps a better tooltip? Should we add some text to the dialog showing which types of URLs we support?

Reply:
Ahh yeah, I forgot that thank you for noticing - The user can't enter an invalid URL right now We show an error if it happens so I am not sure If we need to show which types of URLs we support - besides If they enter a valid git URL we only download files in the default main branch - Do you see value to support also other branches rather than the default main one? It is easy to add

Reply:
i think we should add branch support if it is easy - and some kind of text/placeholder to the form to indicate what URL(s) we support